src/client/index.ts

```diff
@@ -276,7 +277,7 @@ export class Client {
 
 			let err = this.config.unSupportedUserAgent[type].errorMessage;
 
-			if (this.config.type === 'active') {
+			if (this.activeNegotiateRequired()) {
```

Comment:
@oleggrib I don't like this. active/passive should always be a user option. Passive = no-UI (we should probably change the name to suit) as there are some cases where the user wants to implement their own UI. Apart from UI there is really no difference between passive & active.

Reply:
In case of safari/brave/iOS_FF current flow cant be passive, because it requires USER ACTION to open new tab.
If we want to make it really passive and avoid actions then we have to change flow to request tokens list with REDIRECT_FLOW @micwallace @nicktaras 
what do you think?

Reply:
Hi guys, 

I agree with Miccy on this that we should support the passive flow on all browsers over time.

However if I understand correctly this feature is not currently available for full attestation in any browsers accept for chrome. However all browsers can learn attestations via passive.

What are your thoughts that we remove the full attestation support from passive for the time being? Including the passive example hotel bogota?

Thanks.

Reply:
@oleggrib passive mode can be used in the browsers you mentioned, I have done this for hotel Bogota passive and ticket issuer examples. It just requires that Webster handles the POPUP_BLOCKED exception to show a button that re-triggers negotiate when clicked.

Reply:
@nicktaras this is not related to authentication, just negotiate.

Reply:
@micwallace - thanks. I can see that the tickets can be learnt in the examples currently with passive. It's the full attestation flow that cannot be completed.

Reply:
Great inputs on this. 

The outcome is that Oleh is going to create some changes on this PR;

- By default Safari and Brave will auto-redirect for passive flows to acquire token data
- An option will be added to the lib's API for developers to use this default setting, or use a fall back which will use active when passive can't be done which out a ui. This fall back and area can be looked at as a work in progress, where the navigation between tabs will be a key area for improvement for the foreseeable future until we can smooth this flow with less barriers across devices/wallets

This solution will work and enable the flow for devs that use passive flow. Thanks again for the inputs and time put into this.